src/features/profile/Profile.tsx

```diff
@@ -0,0 +1,186 @@
+import { FC } from 'react';
+import { Link, useLocation, useParams } from 'react-router-dom';
+import { Button } from '../../common/components';
+import NarrativeList from '../../common/components/NarrativeList/NarrativeList';
+import {
+  useAppDispatch,
+  useAppSelector,
+  usePageTitle,
+} from '../../common/hooks';
+import { authUsername } from '../auth/authSlice';
+import PageNotFound from '../layout/PageNotFound';
+import {
+  cache,
+  clearError,
+  error,
+  loadProfile,
+  selectUsername,
+  usernameSelected,
+} from './profileSlice';
+import classes from './Profile.module.scss';
+
+export const ProfileNarrativesMessage: FC<{
+  realname: string;
+  yours: boolean;
+}> = ({ realname, yours }) => {
+  if (yours) {
+    return <span>This table shows all of your narratives.</span>;
+  }
+  return (
+    <span>
+      This table shows all narratives owned by {realname} which are also
+      accessible to you.
+    </span>
+  );
+};
+
+export const ProfileResume: FC = () => {
+  return <div>Profile Resume</div>;
+};
+
+export const NarrativesView: FC<{ realname: string; yours: boolean }> = ({
+  realname,
+  yours,
+}) => {
+  return (
+    <div className={classes.narratives}>
+      <ProfileNarrativesMessage realname={realname} yours={yours} />
+      <NarrativeList
+        items={[]}
+        itemsRemaining={0}
+        hasMoreItems={false}
+        loading={false}
+        onLoadMoreItems={() => {}} //eslint-disable-line @typescript-eslint/no-empty-function
```

Comment:
`onLoadMoreItems` and `onSelectItem` were made deliberately optional in case they didn't need to be used, so you don't need to add the empty functions and the eslint disables.

Reply:
Thanks! I will make this update in the next iteration.